usr/share/rear/finalize/GNU/Linux/260_rename_diskbyid.sh

```diff
@@ -2,35 +2,42 @@
 #
 # We call sed once for each substituation
 # it would be better to build one sed script and use this later
-# (like verify/GNU/Linux/21_migrate_recovery_configuration.sh
-#   and finalize/GNU/Linux/150_migrate_disk_devices_layout.sh)
+#  (like finalize/GNU/Linux/250_migrate_disk_devices_layout.sh)
 #
 # OLD_ID_FILE contains entries like these (last 2 lines are multipath targets)
-# cciss-3600508b100104c3953573830524b0004 cciss/c0d0
-# cciss-3600508b100104c3953573830524b0004-part1 cciss/c0d0p1
-# cciss-3600508b100104c3953573830524b0004-part2 cciss/c0d0p2
-# cciss-3600508b100104c3953573830524b0004-part3 cciss/c0d0p3
-# cciss-3600508b100104c3953573830524b0004-part5 cciss/c0d0p5
-# scsi-1HITACHI_770122800061 dm-1
-# scsi-1HITACHI_770122800062 dm-0
+# dm-name-rootvg-lv_swap /dev/mapper/rootvg-lv_swap
+# dm-name-rootvg-lvroot /dev/mapper/rootvg-lvroot
+# dm-uuid-LVM-AkjnD2jS2SCCZKbxkZeaByuZLNdHc6qCYp35NGzNbhMaL3YZYtRxuPoerlAkOcj3 /dev/mapper/rootvg-lvroot
+# dm-uuid-LVM-AkjnD2jS2SCCZKbxkZeaByuZLNdHc6qCeXNhvq6sxq6LQh4aksjr1WUANFuCKooc /dev/mapper/rootvg-lv_swap
+# scsi-0QEMU_QEMU_CD-ROM_drive-scsi0-0-0-0 /dev/sr0
+# virtio-a1bd20bf-f66d-442c-8 /dev/vda
+# virtio-a1bd20bf-f66d-442c-8-part1 /dev/vda1
+# virtio-a1bd20bf-f66d-442c-8-part2 /dev/vda2
+# virtio-a1bd20bf-f66d-442c-8-part3 /dev/vda3
 #
-# Those devices have already been adjusted in
-# verify/GNU/Linux/21_migrate_recovery_configuration.sh
+# cciss-3600508b100104c3953573830524b0004 /dev/cciss/c0d0
+# cciss-3600508b100104c3953573830524b0004-part1 /dev/cciss/c0d0p1
 
-FILES="/etc/fstab /boot/grub/menu.lst /boot/grub2/grub.cfg /boot/grub/device.map /boot/efi/*/*/grub.cfg /etc/lvm/lvm.conf"
+FILES="/etc/fstab /boot/grub/menu.lst /boot/grub2/grub.cfg /boot/grub/device.map /boot/efi/*/*/grub.cfg /etc/lvm/lvm.conf /etc/lilo.conf /etc/yaboot.conf /etc/default/grub_installdevice"
 
-OLD_ID_FILE=${VAR_DIR}/recovery/diskbyid_mappings
-NEW_ID_FILE=$TMP_DIR/diskbyid_mappings
+OLD_ID_FILE="${VAR_DIR}/recovery/diskbyid_mappings"
+NEW_ID_FILE="$TMP_DIR/diskbyid_mappings"
 
 [ ! -s "$OLD_ID_FILE" ] && return 0
 [ -z "$FILES" ] && return 0
 
+# Apply device mapping to replace device in case of migration.
+tmp_layout="$LAYOUT_FILE"
+LAYOUT_FILE="$OLD_ID_FILE"
+source $SHARE_DIR/layout/prepare/default/320_apply_mappings.sh
```

Comment:
Ok, I just wonder if I should add this new function to `lib/layout-functions.sh` or create a new one named `lib/apply-mappings-functions.sh` ... or elsewhere ?